core/src/main/java/nl/nn/adapterframework/pipes/FixedResultPipe.java

```diff
@@ -208,10 +232,8 @@ public void setFileName(String fileName) {
 	}
 
 	/**
-	 * Sets the name of the filename. The filename should not be specified
-	 * as an absolute path, but as a resource in the classpath.
+	 * Name of the file containing the result message. Path is relative to the configuration's root directory (file is expected on the classpath).
```

Comment:
```suggestion
	 * Name of the file containing the result message. 
```
Het pad is inderdaad standaard relatief aan de configuratie root directory. Dat is altijd, overal zo. Ik vind het onhandig dat bij elk attribuut te zetten.